src/main/webapp/index.html

```diff
@@ -0,0 +1,171 @@
+                                        iOS
+                                    </div>  
+                                </div>
+                            </div>
+                        </div>
+                    </div>   
+                </div>
+
+            </div>
+
+            <div class="col-sm-5">
+                <div class="row">
+                    <div class="col" >
+                        <div class="row justify-content-center">
+                            <div class="card text-white">
+                                <div class="card">
+                                    <div class="card-body align-items-center d-flex justify-content-center">
+                                        WEB
+                                    </div>  
+                                </div>
+                            </div>
+                        </div>
+                    </div>   
+                </div>
+
+                <hr>
+
+                 <div class="row">
+                    <div class="col justify-content-center" >
+                            <div class="row">
+                                <div class="card text-white">
+                                        <div class="card">
+                                            <div class="card-body align-items-center d-flex justify-content-center">
+                                                GAMES
+                                            </div>  
+                                        </div>
+                                </div>
+                        </div>
+                    </div>   
+                </div>
+           </div>      
+    </div>
+    
+    <script src="js/script.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+        <!-- Initialize firebase products -->
+        <script src="https://www.gstatic.com/firebasejs/7.15.5/firebase-app.js"></script>
+        <script defer src="https://www.gstatic.com/firebasejs/7.15.5/firebase-auth.js"></script>
+
+        <script>
+        // Your web app's Firebase configuration
+        var firebaseConfig = {
```

Comment:
If you have the auth in index.html, remove the auth.html?

Reply:
I deleted it from the master branch I believe

Reply:
I am still having a problem moving these auth edits to another branch